sdk/openai/Azure.AI.OpenAI/Directory.Build.props

```diff
@@ -21,7 +21,7 @@
   -->
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
   
-  <PropertyGroup>
-    <RequiredTargetFrameworks Condition="$(IsTestProject) != 'true'">netstandard2.0</RequiredTargetFrameworks>
-  </PropertyGroup>
+  <!-- <PropertyGroup>
```

Comment:
Seems like we should just zap this, but I leave it to your discretion.

Reply:
I'm so used to Visual Studio saving all my changes every time I build, that I often forget to manually save my changes whenever I do one-off edits of random text files outside of Visual Studio. Thanks! 🙂